kpops/components/streams_bootstrap_v3/base.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import logging
+import re
+from abc import ABC
+from typing import TYPE_CHECKING
+
+import pydantic
+from pydantic import Field
+
+from kpops.component_handlers.helm_wrapper.model import HelmRepoConfig
+from kpops.components.base_components import KafkaApp
+from kpops.components.base_components.helm_app import HelmApp
+from kpops.components.streams_bootstrap_v3.model import StreamsBootstrapV3Values
+from kpops.utils.docstring import describe_attr
+
+if TYPE_CHECKING:
+    try:
+        from typing import Self  # pyright: ignore[reportAttributeAccessIssue]
+    except ImportError:
+        from typing_extensions import Self
+
+
+STREAMS_BOOTSTRAP_HELM_REPO = HelmRepoConfig(
+    repository_name="bakdata-streams-bootstrap",
+    url="https://bakdata.github.io/streams-bootstrap/",
+)
+STREAMS_BOOTSTRAP_VERSION = "3.0.0"
+STREAMS_BOOTSTRAP_VERSION_PATTERN = r"^(\d+)\.(\d+)\.(\d+)(-[a-zA-Z]+(\.[a-zA-Z]+)?)?$"
```

Comment:
how about enforcing major version 3 like this?
```suggestion
STREAMS_BOOTSTRAP_VERSION_PATTERN = r"^(3)\.(\d+)\.(\d+)(-[a-zA-Z]+(\.[a-zA-Z]+)?)?$"
```

Reply:
It works, but then we will lose the possibility of delivering a clear, easy-to-understand error message to the user. The message would look like this and IMO is harder to figure out what the problem is:

```
1 validation error for StreamsBootstrapV3\nversion\n  String should match pattern '^(3).(\\d+)\\.(\\d+)(-[a-zA-Z]+(\\.[a-zA-Z]+)?)?$' [type=string_pattern_mismatch, input_value='2.1.0', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.6/v/string_pattern_mismatch
```

Reply:
gotcha, then I like your solution with the custom validator better 👍🏻